ydb/core/persqueue/pq_impl.cpp

```diff
@@ -928,6 +928,22 @@ void TPersQueue::AddSupportivePartition(const TPartitionId& partitionId)
     NewSupportivePartitions.insert(partitionId);
 }
 
+NKikimrPQ::TPQTabletConfig TPersQueue::MakeSupportivePartitionConfig() const
+{
+    NKikimrPQ::TPQTabletConfig partitionConfig = Config;
+
+    partitionConfig.MutableReadRules()->Clear();
+    partitionConfig.MutableReadFromTimestampsMs()->Clear();
+    //partitionConfig.MutableConsumerFormatVersions()->Clear();
```

Comment:
uncomment it

Reply:
done
https://github.com/ydb-platform/ydb/pull/1807/commits/2c89b1182d6f8e4cbc77cda3c686f3550c23f3f9